src/rdsn/src/common/backup.thrift

```diff
@@ -20,6 +20,30 @@ include "../../../../idl/dsn.layer2.thrift"
 
 namespace cpp dsn.replication
 
+enum backup_status
+{
+    INVALID,
+    CHECKPOINTING,
+    CHECKPOINTED,
+    UPLOADING,
```

Comment:
if it has no other logic between `CHECKPOINTED` and `UPLOADING`, I think delete `CHECKPOINTED` is ok. checkpoint complete also mean it step to uploading

Reply:
CHECKPOINTED is used for replica server, when all partitions status is CHECKPOINTED, app status will turn into UPLOADING.

Reply:
I mean the checkpointing can be step into uploading directly, not add the checkpointed status

Reply:
CHECKPOINTED is not used for meta server, but is used for replica server. It will be used in further pr.

Reply:
Oh, I see